src/pages/People.js

```diff
@@ -0,0 +1,407 @@
+import React from "react"
+import Helmet from "react-helmet"
+import s from "styled-components"
+import { Grid, Cell } from "styled-css-grid";
```

Comment:
Would prefer you just to write your own css-grid using the built in grid styles and styled-components

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout